app/bundles/PageBundle/EventListener/BuilderSubscriber.php

```diff
@@ -105,33 +109,26 @@ class BuilderSubscriber implements EventSubscriberInterface
     const identifierToken   = '{leadidentifier}';
 
     /**
-     * @param TokenHelper         $tokenHelper
-     * @param IntegrationHelper   $integrationHelper
-     * @param PageModel           $pageModel
-     * @param TranslatorInterface $translator
-     * @param Connection          $connection
-     * @param CorePermissions     $security
-     * @param TemplatingHelper    $templating
-     * @param MauticFactory       $factory
+     * BuilderSubscriber constructor.
+     *
+     * @param CorePermissions   $security
+     * @param TokenHelper       $tokenHelper
+     * @param IntegrationHelper $integrationHelper
+     * @param PageModel         $pageModel
+     * @param BuilderTokey      $builderTokenHelperFactory
      */
     public function __construct(
+        CorePermissions $security,
         TokenHelper $tokenHelper,
         IntegrationHelper $integrationHelper,
         PageModel $pageModel,
-        TranslatorInterface $translator,
-        Connection $connection,
-        CorePermissions $security,
-        TemplatingHelper $templating,
```

Comment:
Are you sure all these dependencies should be removed? I think they are being used in the methods bellow.

Reply:
Im not sure what happened here, maybe a mistake while I was doing the merge conflcits. Anyway, you are right, ill make sure they are added back in.